From ad8a2b996d22e08ef94b130abb6d5be6ec7883ee Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 16:14:16 -0400
Subject: [ethernet] be2net: update UE bit description strings

Message-id: <1410192881-30702-54-git-send-email-ivecera@redhat.com>
Patchwork-id: 92438
O-Subject: [RHEL7.1 PATCH 53/78] be2net: update UE bit description strings
Bugzilla: 1088575
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>

BZ: #1088575

Upstream commit(s):
commit 6bdf8f55d27707ea3f7af0aaddf0e3bb79cf6aed
Author: Vasundhara Volam <vasundhara.volam@emulex.com>
Date:   Thu Jul 17 16:20:25 2014 +0530

    be2net: update UE bit description strings

    This patch updates some description strings for BEx/Skyhawk-R
    UE (unrecoverable error) status register bits. The appropriate strings
    are logged when a UE is detected in the adapter.
    Signed-off-by: Vasundhara Volam <vasundhara.volam@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index 19af5fb..f2b8ee1 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -81,10 +81,10 @@ static const char * const ue_status_low_desc[] = {
  "P1_OB_LINK ",
  "HOST_GPIO ",
  "MBOX ",
- "AXGMAC0",
- "AXGMAC1",
- "JTAG",
- "MPU_INTPEND"
+ "ERX2 ",
+ "SPARE ",
+ "JTAG ",
+ "MPU_INTPEND "
 };
 /* UE Status High CSR */
 static const char * const ue_status_hi_desc[] = {
@@ -109,16 +109,16 @@ static const char * const ue_status_hi_desc[] = {
  "HOST5",
  "HOST6",
  "HOST7",
- "HOST8",
- "HOST9",
+ "ECRC",
+ "Poison TLP",
  "NETC",
- "Unknown",
- "Unknown",
- "Unknown",
- "Unknown",
- "Unknown",
- "Unknown",
- "Unknown",
+ "PERIPH",
+ "LLTXULP",
+ "D2P",
+ "RCON",
+ "LDMA",
+ "LLTXP",
+ "LLTXPB",
  "Unknown"
 };
 
-- 
1.7.1